buildSrc/src/main/kotlin/ktlint-kotlin-common.gradle.kts

```diff
@@ -19,11 +19,10 @@ kotlin {
     }
 }
 
-tasks
-    .withType<KotlinCompile>()
+tasks.withType<KotlinCompile>()
     .configureEach {
         kotlinOptions {
-            apiVersion = "1.4"
+            apiVersion = "1.6"
```

Comment:
```
w: API version 1.4 is deprecated and its support will be removed in a future version of Kotlin
```

Reply:
I don't completely know the consequences of us increasing this apiVersion. Will some plugins etc become incompatible? @Tapchicoma ?

Reply:
@Goooler In [commit](https://github.com/pinterest/ktlint/pull/1449/commits/311b63d6ba7fcc4d3b5bed36c27f4a0089a3b1cd) you have removed the `apiVersion` entirely. What is the reasoning behind this?

Reply:
As the apiVersion is sync with Kotlin version by default.